config/hr_paragraphs.settings.yml

```
@@ -2,5 +2,6 @@ sync_domain: 'http://hrinfo.docksal.site'
 sync_credentials: ''
 csv_source_dir: /var/www/tsv/
 notification_new_user_enabled: false
-reliefweb_api_endpoint: 'https://api.reliefweb.int/v2/reports'
+reliefweb_api_endpoint: 'https://api.reliefweb.int/v1/reports'
```

```suggestion
reliefweb_api_endpoint: 'https://api.reliefweb.int/v2/reports'
```

Missed this one. The v1 is deprecated and will stop returning results "soon-ish".